solana/solana-ibc/programs/solana-ibc/src/execution_context.rs

```diff
@@ -422,7 +422,8 @@ impl ExecutionContext for IbcStorage<'_, '_> {
         seq: Sequence,
     ) -> Result {
         msg!("store_next_sequence_send: path: {path}, seq: {seq}");
-        let store: &mut IbcStorageInner<'_, '_> = &mut self.0.borrow_mut();
+        let store: &mut IbcStorageInner<'_, '_, '_, '_> =
+            &mut self.0.borrow_mut();
```

Comment:
```suggestion
        let store = self.0.borrow_mut();
```
here and throughout.